frontend/src/static/js/v_zoom.js

```diff
@@ -23,6 +37,36 @@ window.vZoom = {
     getSliceLink(slice) {
       return `${window.getBaseLink(this.info.user.repoId)}/commit/${slice.hash}`;
     },
+
+    getNumCommitMessageBody() {
```

Comment:
Rename here also.

Reply:
Can ask senior's opinion.

Reply:
Have changed the naming convention to standardize